src/workers/parser/getSymbols.js

```diff
@@ -47,33 +47,49 @@ function getFunctionParameterNames(path: NodePath): string[] {
 }
 
 function getVariableNames(path: NodePath): SymbolDeclaration[] {
+  let callees = path.node.declarations.reduce((callees, declaration) => {
+    return callees.concat(declaration.init.callee);
+  }, []);
+
+  let args = path.node.declarations.reduce((args, declaration) => {
```

Comment:
I wasn't sure if there was a better way to detect a `require` call but I found that `isVariable` worked and this lead me down this rabbit hole, any ideas if there is a better way or if these reduce calls are a good idea?